packages/pds/src/services/moderation/views.ts

```diff
@@ -0,0 +1,477 @@
+import { Selectable } from 'kysely'
```

Comment:
I don't think moderation views are used anywhere on the pds, so I believe we should be able to leave this file removed 👍